{"error": null, "issues": [{"address": 143, "code": ";\ncontract Lotto {", "debug": "callvalue: 0x38d7ea4c68000\nstorage_1: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\ncalldata_Lotto_0: 0x0\ncalldatasize_Lotto: 0x4\n", "description": "A possible integer overflow exists in the function `fallback`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/temp/The_Ethereum_Lottery-2.sol", "function": "fallback", "lineno": 1, "title": "Integer Overflow ", "type": "Warning"}, {"address": 702, "code": "    owner.transfer(this", "debug": "", "description": "Multiple sends exist in one transaction, try to isolate each external call into its own transaction. As external calls can fail accidentally or deliberately.\nConsecutive calls: \nCall at address: 845\nCall at address: 845\n", "filename": "/temp/The_Ethereum_Lottery-2.sol", "function": "fallback", "lineno": 55, "title": "Multiple Calls", "type": "Information"}, {"address": 1226, "code": ";\ncontract Lotto {", "debug": "callvalue: 0x38d7ea4c68000\nstorage_1: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\ncalldata_Lotto_0: 0x53aab43400000000000000000000000000000000000000000000000000000000\ncalldatasize_Lotto: 0x4\n", "description": "A possible integer overflow exists in the function `buyIn()`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/temp/The_Ethereum_Lottery-2.sol", "function": "buyIn()", "lineno": 1, "title": "Integer Overflow ", "type": "Warning"}, {"address": 1697, "code": "\n  }\n  \n/*\nSelf destruct just in c", "debug": "", "description": "Multiple sends exist in one transaction, try to isolate each external call into its own transaction. As external calls can fail accidentally or deliberately.\nConsecutive calls: \nCall at address: 1697\nCall at address: 1697\nCall at address: 1697\n", "filename": "/temp/The_Ethereum_Lottery-2.sol", "function": "refund()", "lineno": 73, "title": "Multiple Calls", "type": "Information"}, {"address": 1697, "code": "\n  }\n  \n/*\nSelf destruct just in c", "debug": "", "description": "A possible transaction order independence vulnerability exists in function refund(). The value or direction of the call statement is determined from a tainted storage location", "filename": "/temp/The_Ethereum_Lottery-2.sol", "function": "refund()", "lineno": 73, "title": "Transaction order dependence", "type": "Warning"}], "success": true}
